tests/lint/readme.md

```diff
@@ -4,6 +4,72 @@ Our linting process with Flake8 consists of two main steps:
 - Generating a diff
 - Running Flake8 on the diff
 
+## Common problems
+There seem to be several common situations for which the Flake8 errors don't clearly indicate an error free solution.
+
+### Line-breaking functions and statements
+
+Lint errors you may encounter:
+- ET128 (flake8-tabs) *unexpected number of tabs and spaces at start of expression line*
+  - In particular, if this error message "expects x spaces", its likely that it is expecting a
+  code alignment style, rather than hanging indent style.
+  To change this expectation, first ensure that there is a newline after the opening
+  paren/bracket/brace.
+- ET121 (flake8-tabs) *unexpected number of tabs at start of definition line*

```

Comment:
Could you at least give an example of this or refer to the `Preferred formatting`. Most importantly, info I'm currently missing is that double indentation of parameters is desired in order to visually distinguish the funcion parameters from the function body.